README.md

```diff
@@ -1,46 +1,62 @@
-# Getting Started with Create React App and Redux
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+#### Watch-hub
+> Description of the project: 
+This is a web app that displays a collection of anime from [https://ghibliapi.herokuapp.com/](this API). You enter a search text that filters the collection to show your desired result.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template. 
 
-## Available Scripts
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- Major languages: HTML | CSS | JavaScript 
+- Library/Framework: Reactjs | Redux | Jest 
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
+[Live Demo Link](https://deploy-preview-7--vocal-praline-119922.netlify.app/)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Project Video
+[Project Video Link](https://youtu.be/viAMZGRw9d4)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Preview
+![space](./screenshot.png)
 
-### `npm run build`
+## Getting started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+#### Clone this repository
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+```bash
+$ git clone https://github.com/AckonSamuel/watch-hub.git
+```
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+#### cd into directory
+```
+$ cd <relative-path>/watch-hub/
+```
 
-### `npm run eject`
+#### install dependencies
+``` run 
+$ npm i
+```
+#### run project
+```
+$ npm start
+```
```

Comment:
- [ ] I suggest adding one more command to run tests `npm test` 👌
- [ ] Your README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/), kindly add it 👍

Reply:
Sure brother!